config-connector/solutions/iam/kpt/storage-bucket-iam/README.md

```diff
@@ -0,0 +1,36 @@
+Storage Bucket IAM
+==================================================
+# NAME
+  storage-bucket-iam
+# SYNOPSIS
+  Config Connector compatible yaml to enable permissions for a storage bucket.
+# REQUIREMENTS
+  A working Config Connector cluster using a service account with `roles/storage.admin`.
+  A storage bucket managed by IAM.
+  Note: Using uniform bucket-level access control is recommended for this package.
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/storage-bucket-iam .
```

Comment:
Let's be consistent and use `kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/storage-bucket-iam storage-bucket-iam`.